projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -58,6 +58,28 @@ public function init() {
 
 		// Management of publicize (sharing screen, ajax/lightbox popup, and metabox on post screen).
 		add_action( 'post_submitbox_misc_actions', array( $this, 'post_page_metabox' ) );
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'register_assets' ) );
+		add_action( 'enqueue_block_editor_assets', array( $this, 'register_assets' ) );
+	}
+
+	/**
+	 * Register assets.
+	 */
+	public function register_assets() {
+
+		if ( ! wp_script_is( 'jetpack-publicize', 'registered' ) ) {
```

Comment:
Because that method is called twice in the editor - `admin_enqueue_scripts` and `enqueue_block_editor_assets`. So, here we check that if the script is already registered by the hook that fires first.

Reply:
Where does it get called for either of those? The only action registration I see is just above where it's registered for `init`.